src/foundation.cs

```diff
@@ -1776,6 +1784,11 @@ interface NSDateIntervalFormatter {
 
 		[Export ("stringFromDate:toDate:")]
 		string StringFromDate (NSDate fromDate, NSDate toDate);
+
+		[Watch (3,0)][TV (10,0)][Mac (10,12)][iOS (10,0)]
+		[Export ("stringFromDateInterval:")]
+		[return: NullAllowed]
+		string ToString (NSDateInterval dateInterval);
```

Comment:
Going back to fix existing API for XAMCORE_4_0 is a bad use of our limited time this summer. The main issues with `XAMCORE_4_0` are that:

* it's not built, neither on Jenkins or wrench, so it's not tested; and 

* there's no target date (it might never be used) and while we have a hard date for `xcode8`